message-index/messages/GHC-47535/example1/before/NotARecordSelector.hs

```diff
@@ -0,0 +1,9 @@
+module NotARecordSelector where
+
+class Foo a where
+  foo :: a -> ()
+
+data Bar = Bar { bar :: () }
+
+test :: Bar
+test = undefined { foo = () }
```

Comment:
This use of `undefined` makes me nervous. I think it might come across as strange or confusing.

Reply:
I added a brief explanation to the `index.md` about how the `undefined` creates ambiguity concerning the type of the record.